File: app/src/main/java/com/kinwatt/powermeter/ui/ActivityView.java
package com.kinwatt.powermeter.ui;

public interface ActivityView {
    void setSpeed(float speed);
    void setAltitude(double altitude);
    void setDistance(float distance);
    void setPowerAverage3(float power);
    void setPowerAverage5(float power);
    void setPowerAverage10(float power);
}

File: app/src/main/java/com/kinwatt/powermeter/ui/ActivityView.kt
package com.kinwatt.powermeter.ui

interface ActivityView {
    fun setSpeed(speed: Float)
    fun setAltitude(altitude: Double)
    fun setDistance(distance: Float)
    fun setPowerAverage3(power: Float)
    fun setPowerAverage5(power: Float)
    fun setPowerAverage10(power: Float)
}
